spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -186,8 +197,12 @@ def _scores2guesses(self, docs, scores):
 
         return guesses
 
-    def set_annotations(self, docs: Iterable[Doc], batch_tree_ids):
+    def set_annotations(self, docs: Iterable[Doc], activations: ActivationsT):
+        batch_tree_ids = activations["guesses"]
         for i, doc in enumerate(docs):
+            doc.activations[self.name] = {}
+            for activation in self.store_activations:
```

Comment:
Don't we need an `isinstance(self.store_activations, bool)` check here (and in similar locations in the other files)?

Reply:
`store_activations` is a property in `TrainablePipe`, we ensure in the setter that we have the correct data structure:

https://github.com/explosion/spaCy/blob/df513beddb9bc2e2407c15bd0d083df11f56f6bb/spacy/pipeline/trainable_pipe.pyx#L351